devices/updateStatus.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const dynamodb = require('../lib/dynamodb');
+const utils = require('../lib/utils');
+var _ = require('underscore');
+const constants = require('../lib/constants');
+
+module.exports.updateStatus = (event, context, callback) => {
+
+  const input = JSON.parse(event.body);
```

Comment:
Need to validate input here. The chipId and status must not be blank, status must be online or offline value. Return 400 Invalid device data if data error found